docs/resources/elasticsearch.md.erb

```diff
@@ -0,0 +1,126 @@
+---
+title: The Elasticsearch Resource
+---
+
+# elasticsearch
+
+Use the Elasticsearch InSpec audit resource to test the Elasticsearch configuration
+settings. The data is extracted from the live elasticsearch cluster using the curl
+command and default url `http://localhost:9200/_nodes/`. The resource can be called
+with user cridentials, https or http. The resource makes uses of FilterTable and
+its functionality can be used.
+
+## Syntax
+
+An `elasticsearch` resource is called with the elasticsearch params;
+
+    describe elasticsearch(es_ip:'0.0.0.0', es_port:'9200', es_user:'elastic', es_pass:'changeme', https:true, self_signed_cert:true).nodes do
+      its('property') { should include 'value' }
+    end
+
+where
+
+* `es_ip` elasticsearch ip address, default : 0.0.0.0
```

Comment:
We should accept a URI and not an IP/port - see my top-level review comment.

Reply:
Good point. Done.